engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetCapability.java

```diff
@@ -63,4 +64,23 @@ public String toString() { // TODO: used for logging purposes at the moment, inv
 
         return String.format(": capability %s", status);
     }
+
+    public static String getCapabilityName(int capability) {
+        String capabilityName = "N/A";
+        switch (capability) {
+            case GL11.GL_BLEND:
+                capabilityName = "GL_BLEND";
+                break;
+            case GL11.GL_DEPTH_TEST:
+                capabilityName = "GL_DEPTH_TEST";
+                break;
+            case GL11.GL_STENCIL_TEST:
+                capabilityName = "GL_STENCIL_TEST";
+                break;
+            case GL11.GL_CULL_FACE:
+                capabilityName = "GL_CULL_FACE";
+                break;
+        }
+        return capabilityName;
+    }
```

Comment:
In line with previous comment on the same method, this could be handled through a static final [int,String] map, making this method unnecessary.